keyboards/alt67/keymaps/default/keymap.c

```diff
@@ -0,0 +1,211 @@
+#include "alt67.h"
+
+#include "samd51j18a.h"
+#include "i2c_master.h"
+#include "usb/udc.h"
+#include "usb/udi_cdc.h"
+#include "led_matrix.h"
+#include "usb/usb2422.h"
+
+#include "action.h"
+#include "debug.h"
+#include "quantum.h"
+
+enum alt67_keycodes {
+    L_BRI = SAFE_RANGE, //LED Brightness Increase
+    L_BRD,              //LED Brightness Decrease
+    L_PTN,              //LED Pattern Select Next
+    L_PTP,              //LED Pattern Select Previous
+    L_PSI,              //LED Pattern Speed Increase
+    L_PSD,              //LED Pattern Speed Decrease
+    L_T_MD,             //LED Toggle Mode
+    L_T_ONF,            //LED Toggle On / Off
+    L_ON,               //LED On
+    L_OFF,              //LED Off
+    L_T_BR,             //LED Toggle Breath Effect
+    L_T_PTD,            //LED Toggle Scrolling Pattern Direction
+    U_T_AUTO,           //USB Extra Port Toggle Auto Detect / Always Active
+    U_T_AGCR,           //USB Toggle Automatic GCR control
+    DBG_TOG,            //DEBUG Toggle On / Off
+    DBG_MTRX,           //DEBUG Toggle Matrix Prints
+    DBG_KBD,            //DEBUG Toggle Keyboard Prints
+    DBG_MOU,            //DEBUG Toggle Mouse Prints
+};
+
+#define TG_NKRO MAGIC_TOGGLE_NKRO //Toggle 6KRO / NKRO mode
+
+keymap_config_t keymap_config;
+
+const uint16_t keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = KEYMAP(
```

Comment:
Could you replace all of the `KEYMAP`s with `LAYOUT`? 

We've been moving over from one to the other.